packages/fxa-payments-server/server/config/index.js

```diff
@@ -127,6 +127,19 @@ const conf = convict({
       env: 'PAYMENT_TERMS_OF_SERVCIE',
```

Comment:
I noticed there is a typo here -- can we "just" fix it as part of this PR?

Reply:
I cannot say for certain without checking.  I guess that's a no.